packages/cactus-plugin-persistence-fabric/src/test/typescript/integration/fabric-setup-helpers.ts

```diff
@@ -0,0 +1,181 @@
+/**
```

Comment:
Why do we have this file under tests? (I don't see it being consumed by any other test files)

Reply:
it is used in functional tests and we assume that developers might use it during testing and integration process

used in 
cacti\packages\cactus-plugin-persistence-fabric\src\test\typescript\integration\persistence-fabric-functional.test.ts

 88 import {
      enrollAdmin,
      enrollUser,
      getUserCryptoFromWallet,
      } from "./fabric-setup-helpers";

Reply:
it is used in functional tests and we assume that developers might use it during testing and integration process

used in
cacti\packages\cactus-plugin-persistence-fabric\src\test\typescript\integration\persistence-fabric-functional.test.ts

88 import {
enrollAdmin,
enrollUser,
getUserCryptoFromWallet,
} from "./fabric-setup-helpers";